packaging/testbed-server/Cargo.toml.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/Cargo.toml b/Cargo.toml
```

Comment:
I've remove `oxidation/{client,binding}` because they aren't copied in the docker context But without patching, rust will try to find the manifest file and failing to do so